joplin/base/wagtail_hooks.py

```diff
@@ -41,6 +34,14 @@ def before_edit_page(request, page):
     assert request.user.is_authenticated
     print(
         f'BeforeEditHook {request.user.email} is in groups {[group.name for group in request.user.groups.all()]}')
+    print(
+        f'BeforeEditHook {page} is in permission groups {page.view_restrictions.all()}')
+    print(f'{request.user.groups.all()} and {page.view_restrictions.all()[0].groups.all()}')
+    try:
+        assert request.user.groups.all() & page.view_restrictions.all()[0].groups.all()
+    except AssertionError:
+        print('404')
```

Comment:
this is WIP, but i am able to check if the user is in the group that the page view restrictions have